tests/suites/test_suite_ssl.data

```diff
@@ -141,59 +136,49 @@ depends_on:MBEDTLS_SSL_PROTO_TLS1_2
 move_handshake_to_state:MBEDTLS_SSL_IS_CLIENT:MBEDTLS_SSL_CLIENT_KEY_EXCHANGE:1
 
 Test moving clients handshake to state: CERTIFICATE_VERIFY
-depends_on:MBEDTLS_SSL_PROTO_TLS1_2
```

Comment:
Just above (line 136), CLIENT_CERTIFICATE is a TLS 1.3 state on client side as well thus the test should not depend on MBEDTLS_SSL_PROTO_TLS1_2 I'd say. Ditto for FLUSH_BUFFERS (client and server), CLIENT_CERTIFICATE on server side below.

Reply:
`CLIENT_CERTIFICATE` is also used by TLS1.2 . I am not sure what's the problem.